storyblok/storyblok-design-system
# Refactor SbTabPanel and SbTabPanels components to SFC

## Pull request type

Jira Link: [INT-](url)




## How to test this PR



Please navigate to the section titled "With Tab Panels" on the [SbTabs page](http://localhost:6006/?path=/docs/design-system-components-sbicon--default) in Storybook and verify that each tab panel shows the correct content and active state.

## What is the new behavior?



- No new behavior, just refactored code.

## Other information

Please note that I left the SBTabPanels component as a functional component with a render function. I assumed that this was an intentional performance enhancement.
